deepquest/__init__.py

```diff
@@ -403,5 +400,7 @@ def add_dependent_params(parameters):
     parameters['STORE_PATH'] = os.path.join(
         parameters['MODEL_DIRECTORY'], parameters['MODEL_NAME'])
     parameters['DATASET_STORE_PATH'] = parameters['STORE_PATH']
+    parameters['MAX_SRC_INPUT_TEXT_LEN'] = parameters['MAX_INPUT_TEXT_LEN']
+    parameters['MAX_TRG_INPUT_TEXT_LEN'] = parameters['MAX_INPUT_TEXT_LEN']
```

Comment:
Ah yes, sorry. In 832d7c2  I've taken the lines from `build_dataset`:
```python
max_src_in_len=params.get('MAX_SRC_INPUT_TEXT_LEN', None)
if max_src_in_len == None:
    params['MAX_SRC_INPUT_TEXT_LEN'] = params['MAX_INPUT_TEXT_LEN']
max_trg_in_len=params.get('MAX_TRG_INPUT_TEXT_LEN', None)
if max_trg_in_len == None:
    params['MAX_TRG_INPUT_TEXT_LEN'] = params['MAX_INPUT_TEXT_LEN']
```
and moved them to `add_dependent_params` so that these are only set if they aren't specified in the config. Cheers.